tools/src/bin/build-blame.rs

```diff
@@ -306,12 +311,24 @@ fn build_blame_tree(
             }
             Some(ObjectType::Tree) => {
                 let mut entry_builder = blame_repo.treebuilder(None)?;
+                let mut parent_subtrees = Vec::new();
+                for parent_tree in &parent_trees {
+                    let parent_subtree = match parent_tree {
```

Comment:
May be worth a comment that we rely on the `parent_trees` vector to be the same length as blame_parents, for which we need to keep the `None`s in the vector?